virtme/guest/virtme-init

```diff
@@ -440,6 +440,9 @@ if [[ -n "${virtme_graphics}" ]]; then
         pre_exec_cmd=""
     fi
 
+    # Clean up any previous X11 state.
+    rm -f /tmp/.X11*/* /tmp/.X11-lock
```

Comment:
Just to be sure: bash will not complain if the files don't exist because 'rm' is used with '-f', right?

Reply:
correct